fuse-products/src/main/java/software/tnb/product/csb/TomcatCamelSpringBoot.java

```diff
@@ -0,0 +1,56 @@
+package software.tnb.product.csb;
+
+import org.apache.maven.model.Dependency;
+
+import software.tnb.product.LocalProduct;
+import software.tnb.product.Product;
+import software.tnb.product.application.App;
+import software.tnb.product.csb.application.TomcatSpringBootApp;
+import software.tnb.product.customizer.Customizer;
+import software.tnb.product.customizer.component.rest.RestCustomizer;
+import software.tnb.product.integration.builder.AbstractIntegrationBuilder;
+
+import com.google.auto.service.AutoService;
+
+import software.tnb.product.util.maven.Maven;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@AutoService(Product.class)
+public class TomcatCamelSpringBoot extends LocalProduct {
+
+    private TomcatSpringBootApp app;
+
+    @Override
+    public App createIntegrationApp(AbstractIntegrationBuilder<?> integrationBuilder) {
+        // Let's remove restcustomizer, since it exclude tomcat
```

Comment:
I'd say restcustomizer (and all customizers) should be changed to handle the tomcat as well, rather than changing it here